handlers.js

```diff
@@ -79,6 +79,22 @@ async function rank(c, player, num) {
     }
 }
 
+function reply_wait(f)
+{
+    return async function slow_function(reply, args)
+    {
```

Comment:
Same here

Reply:
Resolved in https://github.com/arispoloway/tempus-discord-bot/pull/2/commits/63778ca1eccd7944977fd2ed403f5471db2dd379